test/integration/gateways/pin_pay_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule Gringotts.Integration.Gateways.PinpaymentsTest do
+  # Integration tests for the Pinpayments 
+
+  use ExUnit.Case, async: true
+
+  alias Gringotts.{
+    CreditCard, Address
+  }
+
+  alias Gringotts.Gateways.PinPayments, as: Gateway
+
+  #@moduletag :integration
+
+  @amount Money.new(420, :AUD)
+
+  @bad_card1 %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4100000000000001",
+    year: 2019,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @bad_card2 %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4600000000000006",
+    year: 2019,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @good_card %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4200000000000000",
+    year: 2019,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @add %Address{
+
+    street1: "OBH",
+    street2: "AIT",
+    city: "PUNE",
+    region: "Maharashtra", 
+    country: "IN",
+    postal_code: "411015",
+    phone: "8007810916",
+    
+  }
+
+  @opts [
+    description: "hello", 
+    email: "hi@hello.com",
+    ip_address: "1.1.1.1",
+    config: %{apiKey: "c4nxgznanW4XZUaEQhxS6g", pass: ""}
```

Comment:
Why is `:pass` in config, to best of my memory we removed it from the `required_config`.